### PR TITLE
Added "Are you sure?" modal for FTM JS pause menu items ONLY for language ENGLISH.

### PR DESCRIPTION
**Pull request by Nikhil77sutara**
- Added "are you sure?" modal for level selection and replay level button.

### CODE REVIEW

File: src/scenes/gameplay-scene.ts
Comment (Rajesh1041) [inline src/scenes/gameplay-scene.ts:1]: I don't see any changes in this file!

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  public yesCallback: Function;
+  public noCallback: Function;
+  audioPlayer: AudioPlayer;
+
+  constructor(canvas, yesCallback, noCallback) {
+    this.canvas = canvas;
+    this.yesCallback = yesCallback;
+    this.noCallback = noCallback;
+
+    const selfIdElement = document.getElementById(
```
Comment (Rajesh1041) [inline src/components/feedback-particle-effect/sure-popup.ts:22]: Can we use canvas passed in constructor to get selfIdElement?

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    this.pop_up_image.src = "./assets/images/popup_bg_v01.png";
+    this.pop_up_image.onload = (e) => {
+      this.pop_up_image = this.pop_up_image;
+      this.imagesLoaded = true;
+    };
+  }
+
+  addListner = () => {
+    document
+      .getElementById("canvas")
```
Comment (Rajesh1041) [inline src/components/feedback-particle-effect/sure-popup.ts:54]: Can we use `this.canvas`  here?

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      .addEventListener(CLICK, this.handleMouseClick, false);
+  };
+
+  handleMouseClick = (event) => {
+    const selfElement = <HTMLElement>document.getElementById("canvas");
```
Comment (Rajesh1041) [inline src/components/feedback-particle-effect/sure-popup.ts:59]: Same here!

File: src/components/feedback-particle-effect/sure-popup.ts [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      this.yesButton.draw();
+      this.noButton.draw();
+    }
+  }
+  playClickSound = () => {
+    this.audioPlayer.playButtonClickSound("./assets/audios/ButtonClick.mp3");
+  };
+  dispose = () => {
+    document
+      .getElementById("canvas")
```
Comment (Rajesh1041) [inline src/components/feedback-particle-effect/sure-popup.ts:107]: Same here!

File: src/scenes/gameplay-scene.ts
Comment (Nikhil77sutara) [reply src/scenes/gameplay-scene.ts:1]: Yes we are not having any changes in this file.
I have removed the exrtra lines.

File: src/components/feedback-particle-effect/sure-popup.ts (hunk 1 above)
Comment (Nikhil77sutara) [reply src/components/feedback-particle-effect/sure-popup.ts:22]: yes Rajesh,
i have changed it.

**Review by Rajesh1041 — APPROVED** [review]

**Pull request merged**